### PR TITLE
Manpages

### PR DESCRIPTION
**Pull request by sdgathman**
Fixed capitalization of cjdns.  There are, of course, many more manpages needed.  I mainly wanted to see if cjd et al approve of the markdown -> nroff concept for man pages.

### CODE REVIEW

**Comment by ansuz** [comment]
Looks good to me.

I haven't verified that any of the documented commands work.

Are these specific to the RPM installer? It might be nice to indicate that, if so.

**Comment by kpcyrd** [comment]
I think man pages are a general issue. Looks good to me, I think .md is a good choice.

File: contrib/doc/randombytes.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+=============================================
+
+## SYNOPSIS
+
+`randombytes`
+
+## DESCRIPTION
+
+Writes random bytes to stdout forever using the cryptographically secure
+random generator in libnacl.  In practice, the output is piped
```
Comment (cjdelisle) [inline contrib/doc/randombytes.md]: It's not in libnacl, it's /crypto/Random.c

File: contrib/doc/cjdnslog.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+cjdnslog(1) -- display cjdroute log messages
+=============================================
+
+## SYNOPSIS
+
+`/usr/bin/cjdnslog` [<options>...]
+
+## DESCRIPTION
+
+Cjdroute sends continuous log messages to the bit bucket.  Cjdnslog
```
Comment (cjdelisle) [inline contrib/doc/cjdnslog.md]: teeeechnically the logging never happens unless you subscribe/request the log lines, but however you care to word it is probably fine.

File: contrib/doc/randombytes.md (hunk 1 above)
Comment (sdgathman) [inline contrib/doc/randombytes.md]: Added a commit to change wording.

**Comment by sdgathman** [comment]
ansuz: The commands are the C and nodejs commands included in cjdns.  The cjdns source doesn't install them in any PATH, leaving that up to the system.  The Fedora rpm puts the C and node tools in /usr/libexec/cjdns and symlinks them to /usr/bin.  The symlink renames the tool when there is a conflict with other packages.  E.g. traceroute becomes /usr/bin/cjdns-traceroute.  (Maybe cjdtraceroute would have been better?)

**Comment by ansuz** [comment]
cjdns-traceroute is good, I think.

**Comment by cjdelisle** [comment]
merged to crashey branch, thanks

**Pull request closed without merging**